src/cmd/bundle.go

```diff
@@ -189,3 +207,38 @@ func init() {
 	bundlePullCmd.Flags().StringVarP(&bundleCfg.PullOpts.OutputDirectory, "output", "o", v.GetString(V_BNDL_PULL_OUTPUT), lang.CmdBundlePullFlagOutput)
 	bundlePullCmd.Flags().StringVarP(&bundleCfg.PullOpts.PublicKeyPath, "key", "k", v.GetString(V_BNDL_PULL_KEY), lang.CmdBundlePullFlagKey)
 }
+
+// configureZarf copies configs from UDS-CLI to Zarf
+func configureZarf() {
+	zarfConfig.CommonOptions.Insecure = config.CommonOptions.Insecure
+	zarfConfig.CommonOptions.TempDirectory = config.CommonOptions.TempDirectory
+	zarfConfig.CommonOptions.OCIConcurrency = config.CommonOptions.OCIConcurrency
+	zarfConfig.CommonOptions.Confirm = config.CommonOptions.Confirm
+	zarfConfig.CommonOptions.CachePath = config.CommonOptions.CachePath
+}
```

Comment:
I would make a PR on Zarf to add a `config.SetCommonOptions( opts config.CommonOptions )` function so that this operation is more strongly typed and options are directly tied 1:1 (as it stands someone could accidentally change a line to be like: `zarfConfig.CommonOptions.Confirm = config.CommonOptions.Insecure`)

Reply:
Love it, this was on my list to chat with you all about